src/main/java/ooo/oshi/foreign/windows/Kernel32Library.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributions to this file must be licensed under the Apache 2.0 license or a compatible open source license.
+ */
+package ooo.oshi.foreign.windows;
+
+import java.lang.foreign.*;
+import java.lang.invoke.MethodHandle;
+
+import static java.lang.foreign.ValueLayout.JAVA_INT;
+
+public class Kernel32Library {
+
+    private static final SymbolLookup Kernel32;
+    private static final Linker linker;
+
+    static {
+        linker = Linker.nativeLinker();
+        System.loadLibrary("Kernel32");
+        Kernel32 = SymbolLookup.loaderLookup();
+    }
+
+    private static final MethodHandle getLastError = linker
+            .downcallHandle(Kernel32.lookup("GetLastError").orElseThrow(), FunctionDescriptor.of(ValueLayout.JAVA_INT));
+
+    public static int getLastError() {
```

Comment:
Thanks for adding this, it'll be useful eventually. :). Would like to keep a consistent ordering of putting the user-accessible function first (with full javadoc copied from API) followed by the private MethodHandle.